osvr/RenderKit/RenderManagerD3DBase.cpp

```diff
@@ -703,172 +703,6 @@ namespace renderkit {
       return true;
     }
 
-    bool RenderManagerD3D11Base::ComputeAsynchronousTimeWarps(
```

Comment:
Just to confirm, the reason this method was removed from RenderManagerD3D11Base was that it was modified/moved to the base class?